sql_generators/usage_reporting/templates/composite_active_users_aggregates.view.sql.jinja

```diff
@@ -94,12 +90,10 @@ SELECT
   SUM(monthly_users) AS monthly_users,
 FROM
   `{{ project_id }}.{{ app_name }}.usage_reporting_active_users_aggregates`
-WHERE
-  {% if app_name == "fenix" %}
```

Comment:
@irrationalagent so we do not need any additional filtering here when doing the union?